sdk/communication/AzureCommunicationChat/Source/Options/AzureCommunicationChatClientOptions.swift

```diff
@@ -26,6 +26,11 @@ public struct AzureCommunicationChatClientOptions: AzureClientOptions {
     public let transportOptions: TransportOptions
     /// The default dispatch queue on which to call all completion handler. Defaults to `DispatchQueue.main`.
     public let dispatchQueue: DispatchQueue?
+    /// An array of `PipelineStage` policies to use in lieu of the default ones.
+    public let pipeline: [PipelineStage]?
+    /// An `HTTPTransportStage` policy to use in lieu of the default.
+    public let transport: HTTPTransportStage?
```

Comment:
Can we add these (pipeline, transport) to `TransportOptions` instead? That way they don't have to be repeated in every ClientOptions object and they seem like they make more sense there anyhow.

Reply:
Done.